src/components/util/Rivet.tsx

```diff
@@ -149,9 +149,9 @@ const parseRivetHidden = (hide?: Hidden) => ({
  */
 export type ComponentDecorator<T extends Props> = ( props: T ) => string;
 
-export const rivetize = <T extends React.HTMLAttributes<HTMLElement>>(Component: React.ComponentType<T>): React.SFC<T & Props> =>
+export const rivetize = <T extends React.HTMLAttributes<HTMLElement>>(Component: React.ComponentType<any>): React.FC<any & Props> =>
```

Comment:
👀 This line is particularly complex and the types aren't happy. The only way for me to make it work so far is to changes the types to any.